packages/core/parcel-bundler/src/assets/RawAsset.js

```diff
@@ -27,7 +27,11 @@ class RawAsset extends Asset {
   }
 
   async generateHash() {
-    return await md5.file(this.name);
+    try {
+      return await md5.file(this.name);
+    } catch (err) {
+      throw new Error(err);
```

Comment:
Why do we need to catch and re-throw the error here? Isn't that the same as just not catching it in the first place?